app/views/index.scala.html

```diff
@@ -178,8 +178,19 @@
             <div class="im-centered3">
                 <div class="row" style="width: 450px;">@Messages("landing.ml.title")</div>
                 <div class="row" id="content-text-4"><p>@Messages("landing.ml.content")</p></div>
-                <br>
+            </div>
+            <br>
+            <div class="gif-container">
                 <img id="mlgif" src="@routes.Assets.at("assets/MLGraphic.gif")">
+                <p class="you">@Messages("landing.ml.gif.you")</p>
+                <p class="ml">@Messages("landing.ml.gif.ml")</p>
+                <p class="oc">@Messages("landing.ml.gif.outcomes")</p>
+                <p class="uLabels">@Messages("landing.ml.gif.uLabels")</p>
```

Comment:
For the translations, we may as well just use the full "user.labels" instead of "uLabels". We generally aren't using "camelCase" in the messages file.